jerry-core/debugger/debugger.c

```diff
@@ -202,6 +218,23 @@ jerry_debugger_send_eval (const lit_utf8_byte_t *eval_string_p, /**< evaluated s
 } /* jerry_debugger_send_eval */
 
 /**
+ * Suspend execution for received time
```

Comment:
here, "received" sounds a bit strange to me. as if we would _receive_ it over network (which we don't). alternatives could be:
- Suspend execution for some time.
- Suspend execution for a given time.
- Suspend execution for milliseconds intervals.
- Platform-independent sleep implementation.

The following note might be worth adding anyway: Note: If the platform does not have nanosleep or usleep, this function does not sleep at all.